test/unittests/model/version/test_version_range.py

```diff
@@ -26,6 +27,9 @@
     # Any
     ['*',       [[[">=", "0.0.0-"]]],                  ["1.0", "a.b"],          []],
     ['',        [[[">=", "0.0.0-"]]],                  ["1.0", "a.b"],          []],
+    # Patterns
+    ['1.2.3.*',       [[["*", "1.2.3."]]],   ["1.2.3.1"], ["1.2.4"]],
```

Comment:
What about 1.2.3 itself? I'd expect it to be a matching version, but looks like the `startswith` above will return `False`